src/main/steps/common/form/form.njk

```diff
@@ -23,7 +23,7 @@
             value: true,
             preventDoubleClick: true,
             attributes: {
-              id: 'main-form-submit'
+              id: 'main-form-access-code-check'
```

Comment:
Why is the id changed? Is the id used in code any where? Does it impact any of the functionality? Have you tested the changes?

Reply:
Its duplicated in same file so changed it
Its not used anywhere across the application